decentraland/unity-renderer
# fix: Changed audio clip import settings to minimize size

Many sounds were forced to mono. All sounds brought down to 80% quality.

Issue #1502

## What does this PR change?

Reduces audio asset file sizes from 12.4 MB to 7.0 MB by forcing some sounds into mono and reducing the quality of all assets to 80%.

...

## How to test the changes?

1. Go to [test branch playtest link.](https://play.decentraland.zone/?renderer-branch=fix/audio-asset-size-minimize)
2. Listen to see if the audio still has acceptable quality.
